schedule.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+var minimatch = require('minimatch');
+var nodeSchedule = require('node-schedule');
+
+
+var schedule: { jobs: Array<any>, work: Function, performAt: Function, parse: Function} = {
```

Comment:
nit: either put a space before and after { } or no space at all, but having only a space before and not after is weird